repo/fsrepo/datastores.go

```diff
@@ -366,6 +369,21 @@ func BadgerdsDatastoreConfig(params map[string]interface{}) (DatastoreConfig, er
 		}
 	}
 
+	vls, ok := params["vlogFileSize"]
+	if !ok {
+		c.vlogFileSize = 1 << 30
```

Comment:
Why? Could you add a comment explaining this.

Reply:
It's the default size, will add a comment